gridtools_lib/pylib/make_mosaic/mosaic_util.py

```diff
@@ -0,0 +1,26 @@
+import ctypes
+
+
+
+def get_align_contact():
+
+    #generated shared library from c code
+    #use CDLL function to load the shared library file, returns library object that can be used tto access functions
+    clibrary = ctypes.CDLL('libcontact.so')
+
+    #acquire function signature
+    get_align_contact = clibrary.get_align_contact
+
+    #represent parameters needed
+    get_align_contact.argtypes = [ctypes.c_int, ctypes.c_int, ctypes.c_int, ctypes.c_int, ctypes.c_int, ctypes.c_int, ctypes.POINTER(ctypes.c_double), ctypes.POINTER(ctypes.c_double), ctypes.POINTER(ctypes.c_double), ctypes.POINTER(ctypes.c_double), ctypes.c_double, ctypes.c_double, ctypes.POINTER(ctypes.c_int), ctypes.POINTER(ctypes.c_int), ctypes.POINTER(ctypes.c_int), ctypes.POINTER(ctypes.c_int), ctypes.POINTER(ctypes.c_int), ctypes.POINTER(ctypes.c_int), ctypes.POINTER(ctypes.c_int), ctypes.POINTER(ctypes.c_int)]
```

Comment:
for readability, can we do something like `size1_t = ctypes.c_int` and use that as 
`get_align_contact.argtypes = [size1_t, ... ]`?